syntax/syntax.js

```diff
@@ -739,10 +739,12 @@ function recogniseStuff(scripts) {
         }
         var spec = parts.join(" ")
         var hash = hashSpec(spec)
-        var info = (customBlocksByHash[hash] = {
+
+        var info = {
           spec: spec,
           names: names,
-        })
+        }
+        if (!customBlocksByHash[hash]) customBlocksByHash[hash] = info
```

Comment:
I think this is a change in the order custom blocks are matched? But this shouldn't really matter since this map is only really used for coloring custom blocks anyway, and the end result will be the same either way!